### PR TITLE
added web streaming system test

### PR DESCRIPTION
**Pull request by Manish-210**
zboxcli: https://github.com/0chain/zboxcli/pull/410
gosdk: https://github.com/0chain/gosdk/pull/905

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | add_transcode_test  |
| gosdk  | feature/transcode-on-upload  |
| zboxcli  | feature/web-streaming  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | add_transcode_test  |
| gosdk  | feature/transcode-on-upload  |
| zboxcli  | feature/web-streaming  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | add_transcode_test  |
| gosdk  | feature/transcode-on-upload  |
| zboxcli  | feature/web-streaming  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

File: tests/cli_tests/zboxcli_file_upload_test.go [hunk 1]
```diff
@@ -739,6 +739,155 @@ func TestUpload(testSetup *testing.T) {
...
+		videoFormat := sampleVideo[2]
+		t.RunWithTimeout("Upload Video File "+videoFormat+" With Web Streaming Should Work", 2*time.Minute, func(t *test.SystemTest) {
+			allocSize := int64(400 * 1024 * 1024)
+			allocationID := setupAllocation(t, configPath, map[string]interface{}{
+				"size":   allocSize,
+				"tokens": 9,
+				"expire": "10m",
+			})
+			output, err := cliutils.RunCommand(t, "wget "+videoLink+" -O "+videoName+"."+videoFormat, 3, 2*time.Second)
+			require.Nil(t, err, "Failed to download test video file: ", strings.Join(output, "\n"))
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_file_upload_test.go:877]: Can you abstract this into method?

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]
Minor change but LGTM otherwise

**Review by Kishan-Dhakan — APPROVED** [review]

**Pull request merged**